codecov_cli/services/upload/upload_collector.py

```diff
@@ -112,6 +112,7 @@ def _get_file_fixes(
 
         try:
             with open(filename, "r") as f:
+                lineno = -1
```

Comment:
Maybe a comment here to explain why it starts at -1 and not 0?

Reply:
not sure this actually does anything. `lineno` will be overwritten by whatever `enumerate()` spits out on the next line, right?

is `enumerate(f, start=-1)` what you want?

Reply:
oh i see, we set `eof` outside the loop. could you add a comment before the for loop saying, like, "if not empty, we overwrite lineno. if empty, it stays -1" so nobody deletes your line in the future bc they think it's extraneous